Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/EngineApiWrapper.cs

```diff
@@ -91,6 +96,17 @@ public ScriptableByteArray CrdtGetState()
             }
         }
 
+        [PublicAPI("Used by StreamingAssets/Js/Modules/EthereumController.js")]
+        public object SendEthereumMessageAsync(int id, string method, string jsonParams)
+        {
+            // TODO: support cancellations by id (?)
+            sendCancellationToken = sendCancellationToken.SafeRestart();
+
+            return api.SendEthereumMessageAsync(method, JsonConvert.DeserializeObject<object[]>(jsonParams), sendCancellationToken.Token)
```

Comment:
Make sure that any JSON operations are executed on the thread pool

Reply:
Cannot do `UniTask.SwitchToThreadPool()`. Im getting this exception:
```
InvalidOperationException: The current SynchronizationContext may not be used as a TaskScheduler.
```

Reply:
`object SendAsync(double id, string method, string jsonParams)` is executed from the background thread - you don't have to worry about it.
Potentially you could switch back to the thread pool before 
```
return new SendEthereumMessageResponse
                    {
                        jsonAnyResponse = JsonConvert.SerializeObject(result),
                    };
```

but you limit the request to only one a time so it's fine